gen/app-gen

```diff
@@ -570,6 +570,7 @@ cp -a "${manifests_dir}" "${temp_dir}/manifests"
 generate_metadata "${temp_dir}/metadata.json"
 
 mender-artifact write module-image \
+    --compression none \
```

Comment:
this maybe considered a breaking change. perhaps we should introduce some flag, which will by default do that, but can be disabled. also: why? (this can be also changed with the extra options passed to the mender-artifact on this line via the `passthrough_args`.

Reply:
> why?

compressing something that's already compressed just adds an overhead, both on the artifact generation side and on the artifact processing side

> a breaking change

I don't think it's a breaking change, the client handles this transparently so both variants of artifacts will work fine. It's just an optimization (or rather a drop of a bad optimization).

Reply:
It is not a breaking change: the interface between the Mender client and the Update Module is the same (from another point of view, as Vratislav says, the client handles this internally and will present the same contents to the Update Module)